src/MonC/LLVM/API/Module.cs

```diff
@@ -112,7 +115,8 @@ public void WriteRelocatable(BinaryWriter writer)
 
             // Otherwise write a giant buffer
             using MemoryBuffer memBuf = WriteBitcodeToMemoryBuffer();
-            writer.Write(memBuf.Bytes);
+            // TODO: Remove ToArray when adoping .net standard 2.1
```

Comment:
Maybe we could put LLVM support in a different project/dll that supports latest standard? Kind of like how Driver and Frontend both target .Net Standard 2.1 only.
I don't forsee anyone (or myself) needing to use LLVM support in Unity..